pkgs/by-name/po/poliedros/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+  glib,
+  desktop-file-utils,
+  appstream-glib,
+  gobject-introspection,
+  libadwaita,
+}:
+let
+  version = "1.0.1";
+in
+python3Packages.buildPythonApplication {
+  pname = "poliedros";
+  inherit version;
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "kriptolix";
+    repo = "Poliedros";
+    tag = "v${version}";
+    hash = "sha256-1lYEsfyl6ckH1TmMLRP+flnm77INiA8ntnGVWnwpLvs=";
+  };
+
+  build-system = [ python3Packages.meson-python ];
```

```suggestion
  build-system = [ meson-python ];
```
meson-python should be imported